src/main/java/frc/robot/RobotContainer.java

```diff
@@ -59,20 +69,33 @@ private void configureBindings() {
       m_Drive.drive(forwardSpeed, rightSpeed, rotatinalSpeed);
     }, m_Drive));
 
-    // Schedule `exampleMethodCommand` when the Xbox controller's B button is pressed,
-    // cancelling on release.
-    //m_driverController.b().whileTrue(m_exampleSubsystem.exampleMethodCommand());
-  }
+    contrJoystick.button(1).onFalse(Commands.run(() -> {
+      m_LaunchMech.launchSpeaker();
+    }, m_LaunchMech));
```

Comment:
you may want to consider `onFalse` vs. `whileTrue` as well as `Commands.run` vs. `Commands.startEnd` - not necessarily that this isn't right - you may find better options with those when testing gets here. but don't let this block you.

Reply:
#32